test/integ/matMul/src/matMul.cpp

```diff
@@ -150,26 +150,20 @@ namespace alpaka
                 TIndex const& n,
                 TIndex const& k,
                 TElem const& alpha,
-                TElem const* const A,
-                TIndex const& lda,
-                TElem const* const B,
-                TIndex const& ldb,
+                alpaka::Accessor<TMemoryHandle, TElem, TIndex, 2, alpaka::ReadAccess> const A,
```

Comment:
You missed the renamings of `TMemoryHandle` here.

Reply:
thx!